yeat/tests/test_bandage.py

```diff
@@ -0,0 +1,35 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2022, DHS. This file is part of YEAT: http://github.com/bioforensics/yeat
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+import pytest
+import subprocess
+from unittest.mock import patch
+from yeat.bandage import bandage
+
+
+def test_checking_bandage():
+    assert bandage.check_bandage()
+
+
+@patch("subprocess.run")
+def test_unable_to_run_bandage(function_mock, capsys):
+    completed_process = subprocess.CompletedProcess(["bandage", "--help"], 1)
+    error_message = "FileNotFoundError: [Errno 2] No such file or directory: 'Bandage'"
```

Comment:
This is error message displayed if the $PATH for Bandage is not set up.

Reply:
You're on the right track with this test, but I think you got a little too clever. You directly set the error message, so you're not *really* testing that this is the error message that gets produced in runtime, you're just checking that it's correctly propagated to the terminal (stdout). In other words, you could change the error message to "r0@$t b33f wh@ck@d00dl3" and the test would still pass: not an encouraging sign. :smiley:

I think the problem you're trying to solve here is how to test the "file not found" warning message when Bandage is installed and YEAT can in fact find it. If that's right, please revisit this test and see if you can find a way to elicit a genuine failure (manipulate the `$PATH` maybe?). If not, please let me know where I'm off.

Reply:
Thanks for the suggestion!

Found a solution to mock the $PATH variable.